theme/stencil/src/util/util.ts

```diff
@@ -1,4 +1,33 @@
 import * as uuid from "uuid";
+import { GrowthBook } from "@growthbook/growthbook";
+
+// Initialize GrowthBook
+export const gb = new GrowthBook({
+    apiHost: "https://cdn.growthbook.io",
+    clientKey: (window as any).growthbook_sdk_key,
+    decryptionKey: (window as any).growthbook_decrypt_key,
+    enableDevMode: (window as any).growthbook_dev_mode,
```

Comment:
Neat. Could we move the actual key values into this file and skip adding in the `<head />` ?

Reply:
I don't have the hugo.Environment variable here, i tried to find a way but i don't know if i can detect the environment somehow and didn't find any examples

Reply:
I tried to poke into this.. it looks like we _build and commit_ the components, so I don't think there is an opportunity to do prod/dev key selection in the component code.